app/assets/stylesheets/responsive/_print_layout.scss

```diff
@@ -1,7 +1,6 @@
 div#content, div#left_column, div.entirebody div#wrapper {
   width: 100%;
   margin: 0;
-  float: none;
```

Comment:
that makes sense in theory but as it's breaking things, taking it out seems fair 👍